revenuecat_examples/purchase_tester/android/settings.gradle

```diff
@@ -20,3 +20,8 @@ if (file(".composite-enable").exists()) {
     includeBuild (path)
 }
 
+includeBuild('/Users/madelinebeyl/git/revcat/purchases-hybrid-common/android')
```

Comment:
oops

Reply:
yeah `hybrid-common` needs to be updated, i left this here for now in case yall wanted to test?

Reply:
oh, ok, that's fine, then. 

I'd usually recommend leaving a `// TODO: remove before merging` or a comment in the PR to make the intent clear